src/os/bluestore/BlueStore.cc

```diff
@@ -17865,13 +17865,35 @@ int BlueStore::_merge_collection(
   return r;
 }
 
+void BlueStore::_trim_slow_op_event_queue(mono_clock::time_point cur_time) {
+  if (!cct->_conf->bluestore_slow_op_event_counter_threshold) {
```

Comment:
maybe add  
ceph_assert(ceph_mutex_is_locked(qlock)); 
to explicitly show this has to be called under the lock?